src/Form/listenForFormSubmission.js

```diff
@@ -1,11 +1,7 @@
-const isApp = require('../autofill-utils')
-
 /**
  * @param {Map<HTMLElement, import("./Form").Form>} forms
  */
 const listenForGlobalFormSubmission = (forms) => {
-    if (!isApp) return
```

Comment:
I moved the check to here instead, https://github.com/duckduckgo/duckduckgo-autofill/blob/6fdda5c01ac7d0a6942339a87f16250cd5c581ff/src/DeviceInterface/InterfacePrototype.js#L184-L187 -> it suits my next PR better since we no longer import config directly.